src/Components/Category/Category.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import axios from "axios";
+import { CategoryCard } from "./CategoryCard";
+
+export const Category = () => {
+  const [category, setCategory] = useState([]);
+  useEffect(() => {
```

Comment:
Error handling is missing here if in some cases categories not able to fetch then you need to show something you can do this in future

Reply:
Yes, I missed that. Thanks